client/src/components/Admin/Login/LoginForm.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import Input from './input';
+import './loginform.css';
+
+ class LoginForm extends Component {
```

Comment:
To keep everything consistent, make functional components instead of class-based components

Reply:
Also `.jsx` extension should be changed to `.js` for consistency throughout the code.